sqlitecluster/SQLite.cpp

```diff
@@ -1284,3 +1278,47 @@ map<uint64_t, tuple<string, string, uint64_t>> SQLite::SharedData::popCommittedT
     _committedTransactions.clear();
     return result;
 }
+
+int SQLite::_wrapSQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int64_t warnThreshold, bool skipInfoWarn) const
```

Comment:
Is the only reason we need a wrapper in order to track `_isFirstQuery`? If so, maybe it would be cleaner to just move the current `SQuery` function into `SQLite.cpp`?